R_scripts/NCR_RNAseq_GeneSetEnrichment_analysis.Rmd

```diff
@@ -0,0 +1,273 @@
+---
+title: "NCR RNAseq - enrichment analysis of gene lists of interest"
+author: "WD, NW"
+description: "This R code is to replicate the analyses, figures and tables from the research article entitled: Multi-insecticide resistant malaria vectors in the field remain susceptible to malathion, despite the presence of Ace1 point mutations, 2021"
+date: "9/28/2020"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+Importing libraries :
+```{r import }
+library(fgsea)
+library(data.table)
+library(plotly)
+library(tidyverse)
+
+# set baseFolder / working directory to a "data" folder where all relevant STables "input_data" are saved as .txt files.
+# baseFolder='~/data/'
+
+```
+
+## 1. Loading up the data
+
+```{r load_gene_list}
+
+geneLists = read.table( paste( baseFolder , 'NCR_annotation_V4_gsea_new4real.txt' , sep = '/' ) , header=T ) 
```

Comment:
# supplementary file S2A Table provides the full annotation list of the 13,299 Anopheles coluzzii Ngousso (ACON) genes based on orthology and blastp search as well as the gene sets used for enrichment analysis (GSEA)